.gitmodules

```diff
@@ -109,8 +109,8 @@
 [submodule "Common/jaspColumnEncoder"]
 	path = Common/jaspColumnEncoder
 	url = https://github.com/jasp-stats/jaspColumnEncoder
-	branch = master
+	branch = qt-6.2-cmake-cpp-17
```

Comment:
@amirmasoudabdol once we approve this PR we could change the branch of `jaspResults` and `jaspColumnEncoder` back to `master` and then first merge:
https://github.com/jasp-stats/jaspResults/pull/11
https://github.com/jasp-stats/jaspColumnEncoder/pull/2

Then locally in this branch checked out `git submodule update --remote && git commit -a && git push` ;)

Reply:
well that messed up flatpak but I think I fixed it again